omero/autogen_docs

```diff
@@ -19,36 +20,37 @@ cat $WORKSPACE/OMERO.server/history.rst >> newfile.rst
 mv newfile.rst omero/users/history.rst
 
 echo "Generating configuration properties page"
-$WORKSPACE/OMERO.server/bin/omero config parse --rst | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/config.rst
+omero config parse --rst # Pre-run to see errors
+omero config parse --rst | sed "s|$WORKSPACE|/home/omero|" > omero/sysadmins/config.rst
 
 echo "Generating ldap setdn usage page"
 mkdir -p omero/downloads/ldap
-(cd $WORKSPACE/OMERO.server && bin/omero ldap setdn -h) > omero/downloads/ldap/setdn.out
+(cd $WORKSPACE/OMERO.server && omero ldap setdn -h) > omero/downloads/ldap/setdn.out
 
 echo "Generating advanced CLI help"
-(cd $WORKSPACE/OMERO.server && bin/omero import --advanced-help) 2> advanced-help.txt || echo "Dumped"
+(cd $WORKSPACE/OMERO.server && omero import --advanced-help) 2> advanced-help.txt || echo "Dumped"
 sed 1,5d advanced-help.txt > omero/downloads/inplace/advanced-help.txt
-(cd $WORKSPACE/OMERO.server && bin/omero import --javahelp) 2> java-help.txt || echo "Dumped"
+(cd $WORKSPACE/OMERO.server && omero import --javahelp) 2> java-help.txt || echo "Dumped"
 sed 1,5d java-help.txt > omero/downloads/cli/help.out
 
 echo "Generating DB script example"
-(cd $WORKSPACE && OMERO.server/bin/omero db script --password secretpassword 2>&1) | sed "s|$WORKSPACE|/home/omero|" > omero/downloads/cli/db-script-example.txt
+(cd $WORKSPACE && omero db script --password secretpassword 2>&1) | sed "s|$WORKSPACE|/home/omero|" > omero/downloads/cli/db-script-example.txt
 
 echo "Generating Web configuration templates"
 # Nginx / WSGI
-$WORKSPACE/OMERO.server/bin/omero web config nginx | sed "s|$WORKSPACE/OMERO.server|/home/omero/OMERO.py|g" > omero/sysadmins/unix/install-web/nginx-omero.conf
-$WORKSPACE/OMERO.server/bin/omero web config nginx-location | sed "s|$WORKSPACE/OMERO.server|/home/omero/OMERO.py|g" | grep -v '^#' > omero/sysadmins/unix/install-web/nginx-location.conf
-$WORKSPACE/OMERO.server/bin/omero web config nginx-location | sed "s|/opt/omero/web|/home/omero|g" | grep '^##' | cut -c3-
+omero config set omero.web.application_server wsgi-tcp
+omero web config nginx | sed "s|$WORKSPACE/OMERO.server|/home/omero/OMERO.py|g" > omero/sysadmins/unix/install-web/nginx-omero.conf
+omero web config nginx-location | sed "s|$WORKSPACE/OMERO.server|/home/omero/OMERO.py|g" | grep -v '^#' > omero/sysadmins/unix/install-web/nginx-location.conf
+omero web config nginx-location | sed "s|/opt/omero/web|/home/omero|g" | grep '^##' | cut -c3-
```

Comment:
Note to self. This will have to be adjusted when we rename ``omero ->omero-web``